ScreenCapturerExample.xcodeproj/project.pbxproj

```diff
@@ -312,6 +313,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				DEVELOPMENT_TEAM = SX5J6BN2KX;
+				ENABLE_BITCODE = YES;
```

Comment:
Strange, I'm sure that Bitcode was disabled for this project. Was it done at another level?